### PR TITLE
feat: improve 1D line broadening filter

### PR DESCRIPTION
**Pull request by jobo322**
- https://github.com/cheminfo/nmrium/issues/3261
- add shape property to peaks added manually

### CODE REVIEW

**Comment by lpatiny** [comment]
First conflict should be solved and then I don't know which issue is solved with this PR.

File: package.json [hunk 1]
```diff
@@ -150,4 +150,4 @@
     "vite": "^5.4.9",
     "vitest": "^2.1.3"
   }
-}
+}
```
Comment (lpatiny) [inline package.json:153]:
```suggestion
}

```

**Comment by lpatiny** [comment]
@jobo322 Did you check why the testcases are not passing ?

**Comment by lpatiny** [comment]
@targos We may need your help to debug the lint-check-types problem.

We have the following error is we run `npx tsc --noEmit --traceResolution`


```
======== Resolving module '../../../data/utilities/getAtomsFromMF.js' from '/Users/lpatiny/git/cheminfo/nmrium/src/component/panels/SummaryPanel/SetMolecularFormulaModal.tsx'. ========
Module resolution kind is not specified, using 'NodeNext'.
Resolving in ESM mode with conditions 'import', 'types', 'node'.
Loading module as file / folder, candidate module location '/Users/lpatiny/git/cheminfo/nmrium/src/data/utilities/getAtomsFromMF.js', target file types: TypeScript, JavaScript, Declaration, JSON.
File name '/Users/lpatiny/git/cheminfo/nmrium/src/data/utilities/getAtomsFromMF.js' has a '.js' extension - stripping it.
File '/Users/lpatiny/git/cheminfo/nmrium/src/data/utilities/getAtomsFromMF.ts' exists - use it as a name resolution result.
======== Module name '../../../data/utilities/getAtomsFromMF.js' was successfully resolved to '/Users/lpatiny/git/cheminfo/nmrium/src/data/utilities/getAtomsFromMF.ts'. ========
/Users/lpatiny/git/cheminfo/nmrium/node_modules/typescript/lib/tsc.js:120986
      throw e;
      ^

RangeError: Map maximum size exceeded
    at Map.set (<anonymous>)
    at checkTypeRelatedTo (/Users/lpatiny/git/cheminfo/nmrium/node_modules/typescript/lib/tsc.js:63276:16)
    at isTypeRelatedTo (/Users/lpatiny/git/cheminfo/nmrium/node_modules/typescript/lib/tsc.js:63188:14)
    at isTypeStrictSubtypeOf (/Users/lpatiny/git/cheminfo/nmrium/node_modules/typescript/lib/tsc.js:62312:12)
```

**Comment by jobo322** [comment]
At the moment open the `general settings` it throws an error:

image

**Comment by jobo322** [comment]
The filters in the `processings` panel can't be removed.

**Comment by hamed-musallam** [comment]
Zones panel preferences are not working anymore

**Pull request merged**